### ISSUE TITLE
Built-in footer support for platform table components as supported in core table components

### ISSUE THREAD

**Issue by Jinata92** [SAP/fundamental-ngx#12286]
#### Is this a bug, enhancement, or feature request?

Enhancement

#### Describe your proposal.
There is footer support for Fundamental Core Table components as documented here(https://sap.github.io/fundamental-ngx/#/core/table#with-footer). However, our application uses Platform Table components, and we have observed that there is no built-in footer support available for these components.

<img width="1186" alt="image" src="https://github.com/user-attachments/assets/04397516-058a-4b09-9649-e0bf56fe1fda">

#### Can you handle that on the application side
No
#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

Angular - 15.2.10
F-NGX - 0.43.40

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@Jinata92 Why can't you use the core table?

**Comment by Jinata92** [comment]
> @Jinata92 Why can't you use the core table?

@droshev According to my project requirements, I need to use the Platform Tree Table with multiple selection mode, as shown here(https://sap.github.io/fundamental-ngx/#/platform/table/advanced). This functionality is not available in the Core Table component.

**Comment by droshev** [comment]
@Jinata92 That may be planned, but only for the latest version.

**Comment by droshev** [comment]
@Jinata92 do you plan migrating to the latest version? If so, then we can think about planning it.

**Comment by Jinata92** [comment]
@droshev Yes, we have plans to migrate to Angular 17 as of now. There is a chance that we will migrate to the latest versions subsequently.

**Comment by droshev** [comment]
@Jinata92 the latest is Angular 19 and soon it will be Angular 20. If we develop this feature it will be on the latest in this case probably Angular 20

**Comment by droshev** [comment]
@mikerodonnell89 do you think we can bring this to the platform table?
